pyproject.toml

```
@@ -9,6 +9,12 @@ fallback_version = "unknown"
 [tool.coverage.run]
 omit = ["*/tests/*", "*pygmt/__init__.py"]
 
+[tool.flakeheaven]
+ignore = ["E203", "E266", "E501", "W503", "F401", "E741"]
+max_line_length = 88
+max-doc-length = 79
```

Try changing dashes to underscores following https://github.com/flakeheaven/flakeheaven/issues/105#issuecomment-1146032641
```suggestion
max_doc_length = 79
```